code/game/gamemodes/cult/runes/cult_mech.dm

```diff
@@ -0,0 +1,62 @@
+/datum/rune/cult_mech
+	name = "cult mech rune"
+	desc = "This rune is used to convert a mech into a cult mech."
+	max_number_allowed = 1
+
+	///How many times it was used
+	var/times_used = 0
```

Comment:
is this a singleton across all cult_mech runes? might be worth indicating that in the DMDoc if so

Reply:
It isn't, also there's only one rune, that can do only one mech, lose it or remove it and you won't have a mech anymore